docs/book/support/best-practices.md

```diff
@@ -0,0 +1,56 @@
+# Best Practices, Recommendations, and Tips
+
+# Recommended Repository Structure
+
+```
+├── pipelines
+│   ├── controller
+│   │   ├── **/*.css
+│   ├── views
+│   ├── model
+│   ├── index.js
+├── steps
+│   ├── css
+│   │   ├── **/*.css
+│   ├── images
+│   ├── js
+│   ├── index.html
+├── steps
+│   ├── css
+│   │   ├── **/*.css
+│   ├── images
+│   ├── js
+│   ├── index.html
+├── Dockerfile
+├── .Dockerignore
+├── gitignore
+└── .zen
+```
+
+# Best Practices
+
+# Best Practices and Tips
```

Comment:
Will these be explained? Ignore if this is just a draft / beginning of this page.